frontend/src/components/molecules/Loading.tsx

```diff
@@ -1,12 +1,17 @@
 import React from "react";
+import Backdrop from "@mui/material/Backdrop";
 import CircularProgress from "@mui/material/CircularProgress";
 
 function Loading() {
   return (
-    <div className="h-screen bg-white">
-      <div className="flex justify-center items-center h-full">
+    <div>
+      <Backdrop
+        sx={{ color: "#fff", zIndex: (theme) => theme.zIndex.drawer + 1 }}
+        invisible
+        open
+      >
```

Comment:
Use MUI Backdrop so that the loading spinner is always centered in the middle of the screen (although tbf our loading times are so fast now idk if people will ever see this component)

Reply:
I think we would need to account for varying internet speeds so it's possible we see faster load times because our internet is great but bad internet speed will leave the user with a blank screen (poor UX)

Reply:
Gotcha. Probably also helps that both our server and client are on localhost, which would explain the blazing fast loading times.